source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -1408,9 +1445,47 @@ def isochrone(self, request, instance):
         # Initialize a context for distributed
         distributed_context = self.get_context()
         request_id = request.get("request_id", None)
+
+        pt_object_origin = None
+        pt_object_destination = None
+        if request.get('origin'):
+            origin_detail = self.get_entrypoint_detail(
+                request.get('origin'), instance, request_id="{}_origin_detail".format(request_id)
+            )
+            if not origin_detail:
+                origin_detail = json_address_from_uri(request.get('origin'))
+                if not origin_detail:
```

Comment:
Can't we do this instead?
```suggestion
        if request.get('origin'):
            origin_detail = self.get_entrypoint_detail(
                request.get('origin'), instance, request_id="{}_origin_detail".format(request_id)
            ) or origin_detail = json_address_from_uri(request.get('origin'))
            if not origin_detail:
```

Reply:
Done